modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerHeaderFragment.kt

```diff
@@ -426,7 +260,25 @@ class PlayerHeaderFragment : BaseFragment(), PlayerClickListener {
                 isBuffering = headerViewModel.isBuffering,
                 theme = headerViewModel.theme,
             )
-            binding.sleep.playIfTrue(headerViewModel.isSleepRunning)
+        }
+    }
+
+    private fun setupShelfComposeView() {
+        binding?.shelfComposeView?.setContent {
```

Comment:
We should apply `DisposeOnViewTreeLifecycleDestroyed` strategy to this view.

Reply:
Applied in 631b3c1